core/templates/pages/exploration-editor-page/modal-templates/exploration-modify-translations-modal.component.html

```diff
@@ -0,0 +1,161 @@
+<div class="modal-header">
+  <h4 class="modal-title">Modify Existing Translations</h4>
+</div>
+<div class="modal-body">
+  <p>
+    *Please update any translations below which are easy to update, and tick the ones you're
+    confident about. Those translations will be saved and shown directly to learners as soon
+    as you save the pending changes to the exploration.
+  </p>
+  <p>Any translations that don't have a check mark will be sent for re-translation.</p>
+  <div *ngIf="contentTranslations && (contentTranslations | keyvalue).length > 0" class="content-translations-container">
+    <div *ngFor="let item of contentTranslations | keyvalue" class="content-translation-block">
+      <div class="content-translation-language">
+        <span>
+          <input type="checkbox" name="" id="">
+          <p>{{ getLanguageName(item.key) }}</p>
+        </span>
+        <i class="fas fa-pencil-alt"></i>
+      </div>
+      <div class="content-translation-value e2e-test-translation-{{ item.key }}">
+        <oppia-rte-output-display [rteString]="item.value.translation" class="translations-rte-output-display">
+        </oppia-rte-output-display>
+      </div>
+    </div>
+  </div>
+  <div *ngIf="contentTranslations && (contentTranslations | keyvalue).length === 0">
+    <p class="no-translations-found-text">No available translations found.</p>
+  </div>
+</div>
+<div class="modal-footer">
+  <button type="button" (click)="cancel()" class="btn modify-translations-cancel-button">Cancel</button>
+  <button type="button" (click)="confirm()" class="btn modify-translations-done-button">Done</button>
+</div>
+
+<style>
+  .modal-header {
```

Comment:
Should we consider moving the CSS to separate CSS files? I think it helps generating the relevant RTL CSS.

Reply:
Can be considerable but I simply went with the standards, if you check the modal-templates directory in which this modal is, this is the standard followed by every single modal.

Reply:
I think styles are currently put in the HTML templates, actually, so I have no issues with how it's done here. Thanks for checking!

Reply:
@seanlip It doesn't actually, neither for this modal nor for anything else in the exploration editor page at all. I don't believe we have RTL support  for such pages unless I'm missing something, same with other non-learner facing pages such as the contributor dashboard or the topics and skills dashboard.

Page with language set to Arabic -
<img width="1728" alt="image" src="https://github.com/oppia/oppia/assets/73329256/789ea3c0-f5d8-41ab-94d4-c84432fd6f4d">

Reply:
Hi @Vir-8, I don't think ".. nor for anything else in the exploration editor page at all" is true, see that the top headers have been inverted on the page. Also, adding Multiple Choice / Item selection interactions have the radio button / checkboxes inverted when you preview them with Arabic language set on the creator page. Note that the checkbox used in the modal in the screenshot isn't actually inverted even though the headers are.

Reply:
Hi @kevintab95 and @seanlip, thanks for looking into it. I wasn't including the top header since that is common for all pages. You're right, the radio buttons are inverted for those specific cases. 

However, I tried moving the CSS to a separate file and it did not generate RTL CSS. All it did was change the margins of the language name header so the checkbox, language name and the indicator are sticking to each other in Arabic.

<img width="609" alt="image" src="https://github.com/oppia/oppia/assets/73329256/26d92183-49dd-424e-8d30-fdf3482c96df">

Reply:
Hm, perhaps the styling in the interaction is a bit different from what we have on the modal (where RTL is not working). Anyway, I took a look at the original PR (https://github.com/oppia/oppia/pull/14375) that introduced RTLCSS and it seems like we currently do it for learner facing pages. So it is OK to leave this as-is. Thanks for checking!